plugins-compat-tester-model/src/main/java/org/jenkins/tools/test/model/PluginCompatTesterConfig.java

```diff
@@ -294,7 +298,9 @@ public void setMavenPropertiesFiles( String mavenPropertiesFile ) {
             } else {
                 System.out.println("Using custom Test JDK home: " + testJDKHome);
             }
-            res.put("jvm", new File(testJDKHome, "bin/java").getAbsolutePath());
+            final String javaCmdAbsolutePath = new File(testJDKHome, "bin/java").getAbsolutePath();
+            res.put("jvm", javaCmdAbsolutePath);
+            res.put("javaVersion", getJavaVersion(javaCmdAbsolutePath));
```

Comment:
such approach will lead to setting up the property in Maven executions. Maybe you want to introduce a new method instead

Reply:
Fixed